portfolio/src/main/webapp/projects_experience.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Jess DeJong - STEP Intern</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <div class="header">
+      <img src="images/Profile_photo_Jess.jpg" alt="Profile photo of Jess" height="250">
+      <h1>Jess's Portfolio</h1>
+      <strong>
+        <a href="#">About Me</a>
+        <a href="#">Projects</a>
+        <a href="#">Experience</a>
+        <a href="#">Hobbies</a>
+        <a href="#">Socials & Contact</a>
+      </strong>
+    </div>
+
+    <div id="content">
+      <h1>Projects</h1>
+      <h2>Spherical Inverter</h2>
+      <div class="speech-bubble">
+        <img src="images/Spherical_Inverter_Screenshot.png" alt="Spherical Inverter Screenshot" width="400" border="1px solid black">
+        <p>
+          A MATLAB-based GUI application that visualizes three-dimensional spherical inversion. 
+          Performs inversion on spheres, cylinders, ellipsoids, custom point sets, and MATLAB 
+          functions passed by the user. 
+          <a href="files/Spherical_Inverter_Research_Report.pdf" target="_blank">Resgarch Report</a>.
```

Comment:
Do you intend for this to be on a new line or a new sentence that directly follow the previous content?

Reply:
New sentence that directly follows the previous line content

Reply:
Should I put that link on the previous line, and then wrap that line from there?